thirdparty/fetch-thirdparty-deps.sh

```diff
@@ -49,10 +49,10 @@ fi
 
 if [ ! -f MaxMind.GeoIP2.dll ]; then
 	echo "Fetching MaxMind.GeoIP2 from NuGet"
-	get MaxMind.Db 1.0.0.0
-	get Newtonsoft.Json 6.0.5
-	get RestSharp 105.0.1
-	get MaxMind.GeoIP2 2.1.0
+	get MaxMind.Db 1.1.0.0
+	get Newtonsoft.Json 6.0.8
```

Comment:
> Unable to resolve dependency 'Newtonsoft.Json (≥ 7.0.1)'.

Please change this accordingly https://www.nuget.org/packages/MaxMind.Db/